src/views/instances/stack-configurator.tsx

```diff
@@ -110,7 +110,9 @@ export const StackConfigurator = forwardRef(function StackConfigurator(
 
     useEffect(() => {
         if (stack) {
-            setStackParameters(toKeyedObject(stack.parameters.filter((parameter) => !parameter.consumed)))
+            const filtered = stack.parameters.filter((parameter) => !parameter.consumed)
+            filtered.sort((a, b) => (a.priority < b.priority ? -1 : 1))
```

Comment:
You can also chain these array methods, `stack.parameters.filter(..).sort(..)`, but that's just aesthetics.